src/implementations/twig/components/fact-figures/fact-figures.story.js

```diff
@@ -94,26 +93,32 @@ const getArgTypes = () => ({
 });
 
 const prepareData = (data, args) => {
-  data.items[0].value = args.value;
-  data.items[0].title = args.title;
-  data.items[0].description = args.description;
-  if (args.icon) {
-    data.items[0].icon.name = args.icon;
+  correctPaths(data);
+  const clone = JSON.parse(JSON.stringify(data));
```

Comment:
i see you are using this approach to clone the data, i would prefer the shorter spread syntax { ...data }, but up to you.